docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>Avoid the word dispatch? I think there was more context related to dispatching before, but now it doesn't make as much sense</p><p>Maybe we don't need the cell at all? Or mention just logcdf, users won't have much needed for moment </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>

Reply:
+1

Reply:
<p>Maybe "A similar function exists for <code>logcdf</code>"? The strategy part was related to the dispatching iirc  </p>

Reply:
<p>Should this have been changed?</p>

Reply:
<p>good catch! I will correct this as your initial suggestion </p>